openwisp_radius/templates/djangosaml2/auth_error.html

```diff
@@ -0,0 +1,24 @@
+{% comment %}
+This template is taken from djangosaml2 with sole purpose of
+updating name space of the URL identifier.
+{% endcomment %}
+
```

Comment:
It should not be hard to extend account/base, see here: https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/accounts/templates/account/login.html